build/assets.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GIT_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+for f in $GIT_ROOT/pages/assets/**/*; do
+  if [ "$f" -nt $GIT_ROOT/pages/static/assets.go ]; then
+    go get -u github.com/jteeuwen/go-bindata/...
```

Comment:
Maybe I am misunderstanding how "go get" works, or the build environment is different somehow?

When I delete both the go-bindata src and executable in the bin from my $GOPATH and run the "go get" command, the script is working for me.